python/pyspark/broadcast.py

```diff
@@ -82,7 +83,8 @@ def dump(self, value, f):
         except pickle.PickleError:
             raise
         except Exception as e:
-            msg = "Could not serialize broadcast: " + e.__class__.__name__ + ": " + e.message
+            msg = ("Could not serialize broadcast: " + e.__class__.__name__ + ": "
```

Comment:
Sure, I was trying to minimize changes but I prefer formatting with `%`.  Switched.